### ISSUE TITLE
GUI: ImportError: No module named indexes.base

### ISSUE THREAD

**Issue by OmerShubi** [Addalin/cameranetwork#28]
In the GUI on pressing 'Query'

```bash
2019-09-24 16:18:43,463 [Thread-1    ] [ERROR]  Uncaught exception in ZMQStream callback
Traceback (most recent call last):
  File "/home/shubi/.conda/envs/cvenv2.7/lib/python2.7/site-packages/zmq/eventloop/zmqstream.py", line 438, in _run_callback
    callback(*args, **kwargs)
  File "/home/shubi/.conda/envs/cvenv2.7/lib/python2.7/site-packages/tornado/stack_context.py", line 277, in null_wrapper
    return fn(*args, **kwargs)
  File "/home/shubi/PycharmProjects/cameranetwork/CameraNetwork/mdp/client.py", line 190, in _on_message
    self.on_message(msg)
  File "/home/shubi/PycharmProjects/cameranetwork/CameraNetwork/client.py", line 163, in on_message
    status, cmd, args, kwds = cPickle.loads(msg[0])
ImportError: No module named indexes.base
```
Possibly related to pandas version mismatch. 
From [stackoverflow](https://stackoverflow.com/questions/36888228/python-read-pickle-importerror-no-module-named-indexes-base): 
*This error can be caused by a version mismatch between the version of pandas used to save the dataframe and the version of pandas used to load it.*

**Comment by Addalin** [comment]
check if it is possible to convert the pkl files and open them with the GUI.
might solve #42

**Comment by OmerShubi** [comment]
Camera was running pandas version 0.19.2, while client had pandas 0.24.2. 

Issue was resolved by installing pandas 0.19.2 on client as well. 

If in the future we'll want to update the pandas version, need to make sure both client and servers run the same pandas version.
